src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -150,6 +151,16 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
             });
     }
 
+    private pageHasLayout() {
+        return this.pageState && this.pageState.layout;
+    }
+
+    private changeContainersHeight() {
+        if (this.pageHasLayout() && this.pageState.state.mode === 0) {
+            setTimeout(() => this.dotEditContentHtmlService.setContaintersSameHeight(this.pageState.layout), 400);
```

Comment:
Why is this timeout for?

Reply:
When `load()` gets called, the DOM is loaded, but the JS that adds `the light-blue` and `container menu` is still being processed, so we need it to get the lastest height

Reply:
Why you didn't do this [here](https://github.com/dotCMS/core-web/blob/c0d0199c1e8e8280f598b27a0e9decbe3671eeb4/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts#L62)?